graylog2-web-interface/src/views/components/aggregationwizard/AggregationWizard.tsx

```diff
@@ -67,13 +67,19 @@ const _onElementCreate = (
   values: WidgetConfigFormValues,
   setValues: (formValues: WidgetConfigFormValues) => void,
 ) => {
-  setValues({
-    ...values,
-    [elementKey]: [
-      ...(values[elementKey] ?? []),
-      {},
-    ],
-  });
+  const aggregationElement = aggregationElementsByKey[elementKey];
+
+  if ('addEmptyElement' in aggregationElement) {
```

Comment:
We can also just do:
```suggestion
  if (aggregationElement?.addEmptyElement) {
```